docs/tutorials/what-is-MIOpen.rst

```diff
@@ -0,0 +1,17 @@
+.. meta::
+   :description: MIOpen is AMD’s deep learning primitives library, which provides highly optimized, and hand-tuned implementations of
+   different operators such as convolution, batch normalization, pooling, softmax, activation and layers for Recurrent Neural
+      Networks (RNNs), used in both training and inference [9].
```

Comment:
Please remove the reference.

Reply:
I did. Let me resend the PR. Here's the final text,

MIOpen is AMD’s deep learning primitives library, which provides highly optimized and hand-tuned implementations of different operators such as convolution, batch normalization, pooling, softmax, activation and layers for Recurrent Neural Networks (RNNs), used in both training and inference. Moreover, MIOpen is fully open source, including all its GPU kernels, complementing AMD’s open source ROCm stack. MIOpen is the first to extend the open source advantage into GPU vendor libraries thereby, continuing to embark on the same ethos as the deep learning community.